tests/test_cli.py

```diff
@@ -140,3 +142,47 @@ def test_inject_column_sort(package_path, column_sort_arg):
 def test_version():
     result = runner.invoke(app, ["version"])
     assert result.exit_code == 0
+
+from click.testing import CliRunner
+from paracelsus.cli import cli
+
+
+
+def test_graph_with_inclusion_regex(package_path: Path):
+    result = runner.invoke(
+        app,
+        [
+            "graph",
+            "example.base:Base",
+            "--import-module",
+            "example.models",
+            "--python-dir",
+            str(package_path),
+            "--include-tables",
+            "^com.*",
+        ],
+    )
+    assert result.exit_code == 0
+    assert "regular_table {" not in result.stdout
+    assert "first_test {" in result.stdout
+    assert "second{" not in result.stdout
+
+
+def test_graph_with_exclusion_regex(package_path: Path):
+    result = runner.invoke(
+        app,
+        [
+            "graph",
+            "example.base:Base",
+            "--import-module",
+            "example.models",
+            "--python-dir",
+            str(package_path),
+            "--exclude-tables",
+            "pos.*",
+        ],
+    )
+    assert result.exit_code == 0
+    assert "regular_table {" in result.stdout
+    assert "first {" not in result.stdout
+    assert "second {" in result.stdout
```

Comment:
In this case we have three tables- `comments`, `users`, and `posts`. Since we're trying to exclude the pattern `^pos.*` we expect the comments and users table to be in the diagram but not the posts one.

```python
assert "comments {" in result.stdout
assert "users {" in result.stdout
assert "post {" not in result.stdout
```